dashboard/app/reporting_email.go

```diff
@@ -552,4 +570,30 @@ func appURL(c context.Context) string {
 	return fmt.Sprintf("https://%v.appspot.com", appengine.AppID(c))
 }
 
+func separateCc(to, cc []string) []string {
+	mb := make(map[string]bool, len(to))
+	newCc := []string{}
+	for _, x := range to {
+		mb[x] = true
+	}
+	for _, x := range cc {
+		if _, found := mb[x]; !found {
```

Comment:
`false` is the default value in a map of bools.  So this can just be `if !toSet[email]`